lib/Connector/PixabayConnector.php

```diff
@@ -221,10 +226,6 @@ public function onLibraryProvider(LibraryProviderEvent $event)
                             $result->picture_id,
                             'https://i.vimeocdn.com/video/pictureId_960x540.png'
                         );
-                    } else {
```

Comment:
I am not clear why we moved from using the medium thumbnail with tiny thumbnail fallback, to using the thumbnail for each size of video. Aren't the "large" ones going to be too big for our usage of a thumbnail?

Reply:
Thanks for this! I was thinking that there must be some sort of consistency (ie if we use large video, then we should use large thumbnail as well) haha 😄 but you're right, some thumbnails might be too large for our use case. 

Reverted the changes :)